integration_test/src/test_runner/mod.rs

```diff
@@ -110,38 +117,14 @@ impl fmt::Display for TestDef {
     }
 }
 
-/// load_file_paths reads the given directory and returns a list its file path
-/// names.
-pub fn load_file_paths(dir: PathBuf) -> Result<Vec<String>, Error> {
-    let mut paths: Vec<String> = vec![];
-    let entries = fs::read_dir(dir).map_err(|e| Error::InvalidDirectory(format!("{:?}", e)))?;
-    for entry in entries {
-        match entry {
-            Ok(de) => {
-                let path = de.path();
-                if (path.extension().unwrap() == "yml") || (path.extension().unwrap() == "yaml") {
-                    paths.push(path.to_str().unwrap().to_string());
-                }
-            }
-            Err(e) => return Err(Error::InvalidFilePath(format!("{:?}", e))),
-        };
-    }
-    Ok(paths)
-}
-
-/// parse_test_file_yaml deserializes the given YAML file into a
-/// IntegrationTest struct.
-pub fn parse_test_file_yaml(path: &str) -> Result<IntegrationTest, Error> {
-    let f = fs::File::open(path).map_err(|e| Error::InvalidFile(format!("{:?}", e)))?;
-    let integration_test: IntegrationTest =
-        serde_yaml::from_reader(f).map_err(|e| Error::CannotDeserializeYaml(format!("{:?}", e)))?;
-    Ok(integration_test)
-}
-
 /// integration_test runs the query and function tests contained in the TEST_FILE_DIR directory
 #[test]
 #[ignore]
-pub fn integration_test() -> Result<(), Error> {
+pub fn integration_test() -> Result<()> {
```

Comment:
Since this is arguably the "entry point" of this file, I moved it up to be the first function. That is why you see a few functions on the left-hand side of your screen removed and re-added below.